.pipelines/e2e.yaml

```diff
@@ -13,6 +13,8 @@ pr:
     - pkg/agent
     exclude:
     - e2e/windows
+    - pkg/agent/datamodel/sig_config*.go # SIG config changes
```

Comment:
might also want to add sig_config_test.go here

Reply:
wildcard should take care of it right?